sky/data/storage.py

```diff
@@ -312,6 +310,68 @@ def mount_command(self, mount_path: str) -> str:
         """
         raise NotImplementedError
 
+    def csync_command(self,
+                      csync_path: str,
+                      interval_seconds: Optional[int] = None) -> str:
+        """Returns command to mount CSYNC with Storage bucket on CSYNC_PATH.
+
+        Args:
+          csync_path: str; Path to continuously sync the bucket to.
+          interval_seconds: int; runs the sync command every interval_seconds
+        """
+        store_type = StoreType.from_store(self)
+        store_type_str = store_type.value.lower()
+
+        # Set default interval_seconds if not provided
+        if interval_seconds is None:
+            interval_seconds = sky_csync.CSYNC_DEFAULT_INTERVAL_SECONDS
+
+        # Set the destination which is either root of the bucket
+        # or subdirectory of the bucket
+        if data_utils.is_cloud_store_url(self.source):
+            if self.source is not None:
+                if isinstance(self.source, (str, Path)):
+                    if store_type == StoreType.S3:
+                        bucket_name, path = data_utils.split_s3_path(
+                            self.source)
+                    elif store_type == StoreType.GCS:
+                        bucket_name, path = data_utils.split_gcs_path(
+                            self.source)
+                    else:
+                        with ux_utils.print_exception_no_traceback():
+                            raise exceptions.StorageSpecError(
+                                f'Currently, {store_type} does not support '
+                                'CSYNC mode.')
+                    if path:
+                        destination = '/'.join([bucket_name, path])
+                    else:
+                        destination = bucket_name
+                elif isinstance(self.source, list):
+                    raise TypeError(
+                        'CSYNC mode does not supprot multiple sources '
+                        'for a single storage.')
+        else:
+            assert self.bucket is not None, (
+                'Bucket should be initialized before calling csync_command')
+            destination = self.bucket.name
+
+        # Parse destination to get sync_point for log file naming
+        # The exact mounting point is either the name of bucket
+        # or the subdirectory in it
+        result = urllib.parse.urlsplit(destination)
+        sync_point = result.path.split('/')[-1]
+        log_file_name = f'csync_{store_type_str}_{sync_point}.log'
+        log_path = f'~/.sky/{log_file_name}'
+        install_cmd = mounting_utils.get_redirect_mount_install_cmd()
+        csync_cmd = (f'python -m sky.data.sky_csync csync {csync_path} '
+                     f'{store_type_str} {destination} --interval-seconds '
+                     f'{interval_seconds} --no-follow-symlinks')
+        return mounting_utils.get_mounting_command(StorageMode.CSYNC,
```

Comment:
```suggestion
        return mounting_utils.get_mounting_command(mount_mode=StorageMode.CSYNC,
```
nit: use kwargs for literals. could consider to move this argument at the end to use kwargs